src/app/json-editor/json-editor.component.ts

```diff
@@ -1,6 +1,7 @@
 import { HttpClient } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
 import { NgbModal, NgbModalConfig } from '@ng-bootstrap/ng-bootstrap';
+import { error } from '@angular/compiler/src/util';
```

Comment:
Remove unsued import

Reply:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/46564815/89428118-aba50000-d759-11ea-94a1-8205bf6baff9.png)
if we remove this import error code then found an error. otherwise, it's working fine.

Reply:
Removed unused code.